website/docs/deploy/configuring-unleash.md

```diff
@@ -191,43 +198,40 @@ The logger interface with its `debug`, `info`, `warn` and `error` methods expect
 
 ## Database configuration
 
-:::info
-In-code configuration values take precedence over environment values: If you provide a database username both via environment variables and in code with the Unleash options object, Unleash will use the in-code username.
-:::
+:::info In-code configuration values take precedence over environment values: If you provide a database username both via environment variables and in code with the Unleash options object, Unleash will use the in-code username. :::
```

Comment:
Putting this all on one line breaks the rendered formatting. The final ::: need to be on their own line. I suggest we revert the changes for this paragraph.

Reply:
fixing